modules/installation-approve-csrs.adoc

```diff
@@ -32,37 +32,34 @@ these CSRs are approved or, if necessary, approve them yourself.
 
 . Confirm that the cluster recognizes the machines:
 +
-ifdef::ibm-z[]
-----
-# oc get nodes
-
-NAME 			        STATUS  ROLES    AGE   VERSION
-master-0.cl1mstr0.example.com   Ready   master   20h   v1.14.6+888f9c630
-master-1.cl1mstr1.example.com   Ready   master   20h   v1.14.6+888f9c630
-master-2.cl1mstr2.example.com   Ready   master   20h   v1.14.6+888f9c630
-worker-0.cl1wrk00.example.com   Ready   worker   20h   v1.14.6+888f9c630
-worker-1.cl1wrk01.example.com   Ready   worker   20h   v1.14.6+888f9c630
-----
-endif::ibm-z[]
-ifndef::ibm-z[]
 ----
 $ oc get nodes
```

Comment:
We're now creating one code block for the command itself and a separate code block for its output: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks

That style update came with the introduction of the copy/paste clipboard feature on docs.openshift.com